res/css/views/settings/tabs/user/_HelpUserSettingsTab.pcss

```diff
@@ -31,5 +31,6 @@ limitations under the License.
 }
 
 .mx_CopyableText {
```

Comment:
Given this is global scoped, won't it affect other instances of the component? Shouldn't it be nested in the block above?

Reply:
Thanks @t3chguy, Yes, You're right I forgot to take a case that `.mx_CopyableText` can also be used by other components. I have done amendments accordingly. please check, Thanks for the insight!